File: src/main/java/model/equipment/armor/Helm.java
package model.equipment.armor;

import model.support.allEnums.ArmorMaterial;
import model.support.allEnums.ArmorType;
import model.support.allEnums.EquipmentType;

public class Helm extends Armor {
    public Helm(ArmorMaterial material, double weight, int quality) {
        super(EquipmentType.ARMOR, ArmorType.HELM, material, weight, quality);
    }
}

File: src/main/java/model/equipment/armor/Helm.kt
package model.equipment.armor

import model.support.enums.ArmorMaterial
import model.support.enums.ArmorType.HELM
import model.support.enums.BodyPart
import model.support.enums.BodyPart.HEAD
import model.support.enums.EquipmentType.ARMOR

class Helm(material: ArmorMaterial, weight: Double, quality: Int) : Armor(ARMOR, HELM, material, HEAD, weight, quality)